internal/outgoing/dispatcher.go

```diff
@@ -255,6 +252,15 @@ func (d *Dispatcher) sendEvent(evt models.IncomingEvent, destination models.Dest
 		return
 	}
 
+	if destination.Config.GetDelayValue() != time.Duration(0) {
+		// Sending message with delay in goroutine, no error will be caught
+		callbackLogger.Info(fmt.Sprintf("Event delayed with %d", destination.Config.Delay))
+		go time.AfterFunc(destination.Config.GetDelayValue(), func() {
+			sender.SendEvent(customizedEvt, destination)
+		})
+		return
```

Comment:
do we need to add the "Event sucessfully sent to ..." logging  inside the subroutine?
like L274
and the error handling.
```
callbackLogger.Info(fmt.Sprintf("Event successfully sent to %s [%s]", destination.Config.Name, destination.GetCallbackURL()))
```